_maps/map_files/SolitaireStation/Solitairestation.dmm

```diff
@@ -22535,6 +22537,19 @@
 /obj/machinery/holopad,
 /turf/open/floor/iron,
 /area/tcommsat/server)
+"giX" = (
+/obj/machinery/door/airlock/medical{
+	name = "Morgue";
+	req_one_access_txt = "6;5;28"
```

Comment:
```suggestion
	req_one_access_txt = "6"
```
Chefs have morgue access

Reply:
ya sure dude this is what i get for using helio sation as a refrnce